cms/envs/common.py

```diff
@@ -358,6 +358,9 @@
 # Clickjacking protection can be enabled by setting this to 'DENY'
 X_FRAME_OPTIONS = 'ALLOW'
 
+# Platform for Privacy Preferences header
+P3P_HEADER = 'CP="Open EdX does not have a P3P policy."'
```

Comment:
FWIW that article was written in 2012 and I think things have evolved since then. From the research I have done, P3P seems to be a dead standard that no except for IE implements. @smagoun Is there someone in legal who can/should look at this?